xbmc/video/VideoDatabase.cpp

```diff
@@ -12214,7 +12214,8 @@ int CVideoDatabase::GetVideoVersionInfo(int idFile,
     if (m_pDS->num_rows() > 0)
     {
       idVideoVersion = m_pDS->fv("id").get_asInt();
-      typeVideoVersion = m_pDS->fv("name").get_asString();
+      typeVideoVersion = +idVideoVersion ? g_localizeStrings.Get(idVideoVersion)
```

Comment:
is the `+` in front of  `+idVideoVersion` a typo?

Reply:
it is.